packages/utilities/src/DelayedRender.tsx

```diff
@@ -43,7 +43,7 @@ export class DelayedRender extends React.Component<IDelayedRenderProps, IDelayed
   constructor(props: IDelayedRenderProps) {
     super(props);
     this.state = {
-      isRendered: false,
+      isRendered: typeof window === 'undefined',
```

Comment:
Rather than access `window` directly you should use the [`getWindow()` utility function](https://github.com/microsoft/fluentui/blob/master/packages/utilities/src/dom/getWindow.ts#L22).

Reply:
Thanks for the review. I've updated the PR to use `getWindow()`.